LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2019 Manifold
+Copyright (c) 2024 Netherlands eScience Center
```

Comment:
Not sure how much of the code Manifold wrote is still there, but dropping them seems not nice.

```suggestion
Copyright (c) 2019 Manifold, 2024 Netherlands eScience Center
```